RELEASE-NOTES.txt

```diff
@@ -1,14 +1,16 @@
 22.2
 -----
-
 * [**] [Jetpack-only] Added a dashboard card for purchasing domains. [#20424]
+* [**] [internal] Refactored Google SignIn implementation to not use the Google SDK [#20128]
 
 22.1
 -----
 * [**] [internal] Refactor updating account related Core Data operations, which ususally happens during log in and out of the app. [#20394]
 * [***] [internal] Refactor uploading photos (from the device photo, the Free Photo library, and other sources) to the WordPress Media Library. Affected areas are where you can choose a photo and upload, including the "Media" screen, adding images to a post, updating site icon, etc. [#20322]
 * [**] [WordPress-only] Warns user about sites with only individual plugins not supporting core app features and offers the option to switch to the Jetpack app. [#20408]
-* [**] [internal] Refactored Google SignIn implementation to not use the Google SDK [#20128]
```

Comment:
As mentioned in the PR description, this entry was in 22.1 because that's where the PR was meant to land after code freeze but I fumbled and it landed on `trunk` instead.